### PR TITLE
fix circle-ci

### PR DESCRIPTION
**Pull request by NicolasDP**

### CODE REVIEW

**Comment by NicolasDP** [comment]
@mzabaluev could you have a look on the error here ? looks related to the network.

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,6 +36,7 @@ jobs:
       - run:
           name: Print version information
           command: rustc --version; cargo --version
+      - run: git submodule update --init --remote
```
Comment (mzabaluev) [inline .circleci/config.yml]: It's better to put this right after the checkout step.

File: .circleci/config.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ jobs:
     working_directory: /mnt/crate
     steps:
       - checkout
+      - run: git submodule update --init --remote
```
Comment (mzabaluev) [inline .circleci/config.yml]: See also the [recommended way to do it](https://discuss.circleci.com/t/can-not-get-submodules-to-clone-properly/22120) accordingly to CircleCI support staff.

**Comment by mzabaluev** [comment]
@NicolasDP a local checkout of jormungandr master builds for me if submodules are checked out accordingly to the [documentation](https://circleci.com/docs/2.0/configuration-reference/#checkout).

File: .circleci/config.yml (hunk 2 above)
Comment (mzabaluev) [reply .circleci/config.yml]: Or a slightly different [recipe](https://circleci.com/docs/2.0/configuration-reference/#checkout) in the reference doc.

**Comment by mzabaluev** [comment]
I think the `--remote` option breaks it, as it checks out the tip of the submodule's branch, which is often too advanced for jormungandr and we want to keep it pinned to a particular commit in rust-cardano.

**Comment by mzabaluev** [comment]
Works now.

**Pull request merged**